extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
+import {Actions} from './actions';
```

Comment:
Please update PR description to be more specific. This PR looks like parses and instruments actions for a local platform. Right?

Reply:
Yep, this one does not start the execution yet